packages/block-editor/src/store/actions.js

```diff
@@ -192,6 +192,17 @@ export function multiSelect( start, end ) {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that the block selection is cleared.
```

Comment:
> Returns an action object used in signalling that the block selection is cleared.

Both `clearSelectedBlock` and `wipeSelectedBlock` have the same description. It should be further clarified so developers could decide which one fits better to their use case.